recipe/meta.yaml

```diff
@@ -28,17 +28,20 @@ requirements:
     - numpy
     - numpydoc
     - ophyd >=1.5.0
+    - pcdsdevices
     - pcdsutils
-    - pydm >=1.16.0
-    - pyqt >=5
+    - platformdirs
+    - pydm >=1.19.1
+    - pyqt
     - pyqtgraph
+    - pyyaml
     - qdarkstyle
     - qtawesome
-    - qtconsole
     - qtpy
     - timechart >=1.2.4
   run_constrained:
     - happi >=1.14.0
+    - jupyter_client <7.3.2
```

Comment:
Is this used only for the console stuff we removed in this version, or is it used elsewhere too? I'm trying to figure it out

Reply:
Console stuff I'm pretty sure